conventional-release/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function output {
+  echo "::set-output name=${1}::${2}"
+}
+
+flags=(--ci)
+
+if [ "${DRY_RUN}" ]; then
+  flags+=(--dry-run)
+fi
+
+if [ "${DEBUG}" ]; then
+  flags+=(--debug)
+fi
+
+previousVersion=$(git describe --tags --abbrev=0)
+output "previousVersion" "${previousVersion}"
```

Comment:
These outputs are here for the next step -- opening PRs to repos where `lead-terraform` is used.